tensorboard/compat/BUILD

```diff
@@ -44,6 +44,7 @@ py_library(
     deps = [
         ":compat",
         "//tensorboard/compat/tensorflow_stub",
+        "//tensorboard/writer",
```

Comment:
Let's not have this dependency here. Instead it should be in https://github.com/tensorflow/tensorboard/blob/master/tensorboard/BUILD#L45 like

```
    deps = [
        ":lazy",
        ":notebook",
        ":program",
        "//tensorboard/summary",
        "//tensorboard/writer",
    ],
```